src/client/routes/game/Board/Board.tsx

```diff
@@ -1,20 +1,21 @@
-import * as React from "react";
+import * as React from "react"
 import { observer, inject } from "mobx-react"
 import { IRootStore } from "../../../store/AppStore"
 import Cell from "./Cell"
 import { WithStyles, createStyles, withStyles } from "@material-ui/core"
 import BoardStore from "../../../store/BoardStore"
 
-const Board = inject((stores: IRootStore) => ({ boardStore: stores.app.game.boardStore }))(observer((props: IBoardProps) => (
-  <div className={props.classes.board} data-test-ui-component="board">
-    {props.boardStore.cells.map((cell) => (
-      <Cell
-        {...cell}
-        key={cell.position.col + cell.position.row}
-      />
-    ))}
-  </div>
-)))
+const Board = inject((stores: IRootStore) => ({
+  boardStore: stores.app.game.boardStore,
+}))(
+  observer((props: IBoardProps) => (
+    <div className={props.classes.board} data-test-ui-component="board">
+      {props.boardStore.cells.map(cell => (
+        <Cell {...cell} key={cell.position.col + cell.position.row} />
+      ))}
+    </div>
+  ))
+)
 
```

Comment:
Tää on anyways ihan perseest lukee, ja en tiiä helpottaaks prettier tässä. Pitäis ehk muuttaa samaks kun Cell.tsx:ssä